validator/testcases/regex.py

```diff
@@ -363,6 +363,67 @@ def tests(self):
             compat_type="error")
 
 
+@register_generator
+class Gecko40RegexTests(CompatRegexTestHelper):
+    """Regex tests for Gecko 40 updates."""
+
+    VERSION = FX40_DEFINITION
+
+    def tests(self):
+        yield self.get_test(
+            r"\b([gs]etKeywordForBookmark|getURIForKeyword)\b",
+            "The old keywords API is deprecated.",
+            "The old keywords API is deprecated. You should use "
+            "PlacesUtils.keywords instead. See %s for more information."
+            % MDN_DOC % "Mozilla/Tech/Places/Using_the_Places_keywords_API",
+            log_function=self.err.warning,
+            compat_type="warning")
+
+        yield self.get_test(
+            r"\bfuelIApplication\b",
```

Comment:
This is also accessible via the `Application` global in browser windows, and the `extIApplication` interface.